### PR TITLE
[UT03-923] Feat/refining with forked celery export plugin

### PR DESCRIPTION
**Pull request by GergelyKovach**
# Description

Several minor changes as described in [UT03-923](https://pulilab.atlassian.net/browse/UT03-923)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Locally with console email. And locally with  `fab cov` and `fab lint`

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: django/tiip/settings.py [hunk 1]
```diff
@@ -18,9 +18,9 @@
 ALLOWED_HOSTS = ['*']
 
 PROJECT_NAME = env.str('PROJECT_NAME', default='Example')
-DEFAULT_FROM_EMAIL = env.str('DEFAULT_FROM_EMAIL', default='john@example.org')
+DEFAULT_FROM_EMAIL = env.str('DEFAULT_FROM_EMAIL', default='invent@unicef.org')
```
Comment (nullcode) [inline django/tiip/settings.py]: why did you change these? please revert to original. open source should not have any invent reference

**Review by nullcode — CHANGES_REQUESTED** [review]

File: django/tiip/settings.py (hunk 1 above)
Comment (GergelyKovach) [reply django/tiip/settings.py]: changed back
i'll keep in mind opensource-ness

File: django/core/admin/__init__.py [hunk 2]
```diff
@@ -150,9 +150,19 @@ class NewsFeedAdmin(SortableAdminMixin, TranslationAdmin):
     pass
 
 
-# extending the exclude list with 'site_of_origin'
+# extending the exclude list
```
Comment (nullcode) [inline django/core/admin/__init__.py]: remove comment

File: django/core/admin/__init__.py [hunk 3]
```diff
@@ -150,9 +150,19 @@ class NewsFeedAdmin(SortableAdminMixin, TranslationAdmin):
...
 class ExportJobAdminNew(ExportJobAdmin):
-    exclude = ('job_status', 'site_of_origin',)
+    exclude = ('job_status', 'site_of_origin', 'app_label', 'model', )
+
+    readonly_fields = (
+        "job_status_info",
+        "author",
+        "updated_by",
+        # "app_label",
```
Comment (nullcode) [inline django/core/admin/__init__.py]: remove commented lines

**Review by nullcode — CHANGES_REQUESTED** [review]

File: django/core/admin/__init__.py (hunk 2 above)
Comment (GergelyKovach) [reply django/core/admin/__init__.py]: this might be unclear why we redefine the ExportJobAdmin

File: django/core/admin/__init__.py (hunk 3 above)
Comment (GergelyKovach) [reply django/core/admin/__init__.py]: as its an override of default state i feel this is useful if in the future its reactivated

File: django/core/admin/__init__.py (hunk 2 above)
Comment (GergelyKovach) [reply django/core/admin/__init__.py]: resolved: removed the comments

File: django/core/admin/__init__.py (hunk 3 above)
Comment (GergelyKovach) [reply django/core/admin/__init__.py]: resolved: removed the comments

**Review by nullcode — APPROVED** [review]

**Pull request merged**